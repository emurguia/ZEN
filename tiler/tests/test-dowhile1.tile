// This tests that the do while loop works as expected

init {
	int a;
	int b;

	tile(3,3);

	a = 100;
	b = 4;

	do {
		sprint("Mama Mia~");
	} while(a < 1);

	do {
		iprint(b);
		b = b - 1;
	} while (b > 0);

	close();

}