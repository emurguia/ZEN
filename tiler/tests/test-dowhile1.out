Mama Mia~
4
3
2
1
The game window was successfully launched ...
The game window has been exited. Program exited.
Game over.
